base/docker_tags.sh

```diff
@@ -22,8 +26,11 @@ docker_tag_exists() {
 
 docker_image()
 {
-    if docker_tag_exists prestashop/base ${version}; then
-        echo "Docker Image already pushed : prestashop/base:$version"
+    if ! $FORCE; then
+        if docker_tag_exists prestashop/base ${version}; then
+            echo "Docker Image already pushed : prestashop/base:$version"
+            return
+        fi
```

Comment:
You're lacking a use case here, if force is not specified and the docker tag doesn't exist it won't be pushed

Reply:
```suggestion
    if ! $FORCE && docker_tag_exists prestashop/base ${version}; then
        echo "Docker Image already pushed : prestashop/base:$version"
```

Reply:
Maybe something like that instead?

Reply:
Fixed